components/gitpod-protocol/src/util/gitpod-cookie.ts

```diff
@@ -3,7 +3,10 @@
  * Licensed under the GNU Affero General Public License (AGPL).
  * See License-AGPL.txt in the project root for license information.
  */
+
 import * as cookie from "cookie";
+import * as Cookies from "js-cookie";
+import { CookieAttributes } from "js-cookie";
```

Comment:
I ended up using the library `js-cookie` that we are also using in the frontend because this worked better in setting multiple cookies.